src/utils/hub-channel.js

```diff
@@ -196,6 +200,22 @@ export default class HubChannel {
     this.channel.push("unpin", payload);
   };
 
+  fetchPermissions = () => {
+    return new Promise((resolve, reject) => {
+      this.channel
+        .push("refresh_perms_token")
+        .receive("ok", res => {
+          this.setPermissionsFromToken(res.perms_token);
+          resolve({ permsToken: res.perms_token, permissions: this._permissions });
```

Comment:
is the passed `permissions` used here?

Reply:
Ah, no but I imagined it will be used when we want to update more of the UI based on permissions